packages/api/endpoints/executions.js

```diff
@@ -79,6 +86,38 @@ async function del(req, res) {
   return res.send({ message: 'Record deleted' });
 }
 
+/**
+ * Get execution history for a single granule or multiple granules
+ *
+ * @param {Object} req - express request object
+ * @param {Object} res - express response object
+ * @returns {Promise<Object>} the promise of express response object
+ */
+async function history(req, res) {
+  const { granuleId } = req.params;
+  const payload = req.body;
+  const granuleIds = granuleId ? [granuleId] : await getGranuleIdsForPayload(payload);
+
+  const knex = await getKnexClient({ env: process.env });
```

Comment:
```suggestion
  const knex = await getKnexClient();
```
`env` defaults to `process.env` in `getKnexClient`, so I think this is unnecessary